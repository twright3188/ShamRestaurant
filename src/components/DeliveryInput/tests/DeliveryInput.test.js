import React from 'react';
import renderer from 'react-test-renderer';
import { DeliveryInput } from '../index';

it ('delivery input renders correctly', () => {
    const tree = renderer.create(<DeliveryInput attributeName='name' value='test' />).toJSON();
    expect(tree).toMatchSnapshot();
});