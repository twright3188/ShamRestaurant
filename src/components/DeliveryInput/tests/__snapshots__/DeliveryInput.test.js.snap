// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`delivery input renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "borderBottomColor": "rgba(34, 38, 36, 0.15)",
      "borderBottomWidth": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "paddingHorizontal": 15,
      "paddingVertical": 16,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "flexWrap": "wrap",
          "fontFamily": "Helvetica",
          "fontWeight": "400",
        },
        Object {
          "color": "#222624",
          "fontSize": 16,
          "lineHeight": 24,
        },
        Object {},
      ]
    }
  >
    name
  </Text>
  <Text
    style={
      Array [
        Object {
          "flexWrap": "wrap",
          "fontFamily": "Helvetica",
          "fontWeight": "400",
        },
        Object {
          "color": "#222624",
          "fontSize": 16,
          "lineHeight": 24,
        },
        Object {
          "color": "#78807B",
        },
      ]
    }
  >
    test
  </Text>
</View>
`;
